docs/production/mobile-push-notifications.md

```diff
@@ -205,25 +177,22 @@ mobile app can take dozens of hours to set up even for an experienced
 developer, and even more time to maintain. And it doesn't provide
 material privacy benefits -- your organization's push notification
 data would still go through Apple/Google's servers, just not Kandra
-Labs'. Our view is the correct way to optimize for privacy is
-end-to-end encryption of push notifications. But in the interest of
```

Comment:
Why are we dropping mention of E2E?

Reply:
I was just thinking it's not super helpful to mention it here. It just begs the question of, OK, why don't you have E2E, then?

I don't feel super strongly about this.